docs/index.md

```diff
@@ -2,15 +2,18 @@
 :maxdepth: 2
 :titlesonly:
 :caption: General
+:hidden:
 
-
-Overview <self>
 Readme <misc/readme_link>
 User Guide <userguide/userguide>
 Examples <examples/examples>
 How To Contribute <misc/contributing_link>
 ```
 
+```{include} ../README.md
+:relative-docs: docs/
+```
+
 # Package documentation
```

Comment:
Can we please also shorten this to simply `Package` ? I think `Package documentation` unnecessarily sticks out a lot in terms of length from all other topics. At the same time if Im looking for a package code documentation Im pretty sure Id go here if I read only `Package` so we dont loose anything